data_model/src/isi.rs

```diff
@@ -936,50 +936,42 @@ pub mod error {
             Encode,
             IntoSchema,
         )]
+        #[ignore_extra_doc_attributes]
```

Comment:
Why is it needed here? It seems that the doc of `InstructionExecutionError` is a single line already.

If it is needed for `::InvariantViolation` variant, isn't it possible to set the attribute for the variant only?

Reply:
`displaydoc` doesn't support this =(


`#[ignore_extra_doc_attributes]` only works when placed on the whole type